images/alpine.yaml

```diff
@@ -195,6 +195,35 @@ files:
 - path: /etc/hosts
   generator: hosts
 
+- generator: fstab
+  types:
+  - vm
+
+- generator: lxd-agent
+  types:
+  - vm
+
+- path: /etc/default/grub
+  generator: dump
+  content: |-
+    # Set the recordfail timeout
+    GRUB_RECORDFAIL_TIMEOUT=0
+
+    # Do not wait on grub prompt
+    GRUB_TIMEOUT=0
+
+    # Set the default commandline
+    # FIXME: Don't hard-code filesystem
+    GRUB_CMDLINE_LINUX_DEFAULT="console=tty1 console=ttyS0 modules=sd-mod,usb-storage,ext4 rootfstype=ext4"
```

Comment:
We need a way to not hard-code the filesystem. This could be done by using pongo in the dump generator.

Reply:
See https://github.com/lxc/distrobuilder/pull/336